web/packages/teleport/src/Discover/Overview/Overview.tsx

```diff
@@ -0,0 +1,282 @@
+        </InfoUl>
+      );
+      break;
+
+    case DiscoverGuideId.KubernetesAwsEks:
+      links = {
+        accessEntries: {
+          title: 'Using access entries',
+          href: 'https://docs.aws.amazon.com/eks/latest/userguide/setting-up-access-entries.html',
+        },
+      };
+      overview = (
+        <InfoParagraph>
+          This guide is used to set up auto-enrollment of one or more EKS
+          clusters with a script that you will run in AWS CloudShell.
+        </InfoParagraph>
+      );
+      prereqs = (
+        <InfoUl>
+          <li>{requiredAwsAccount}</li>
+          <li>
+            <InfoExternalTextLink href={links.accessEntries.href}>
+              Access entries
+            </InfoExternalTextLink>{' '}
+            authentication mode enabled in your target EKS clusters.
+          </li>
+          <li>
+            List of Kubernetes users and groups you want Teleport users to be
+            able to authenticate as.
+          </li>
+        </InfoUl>
+      );
+      break;
+
+    case DiscoverGuideId.ApplicationAwsCliConsole:
+      overview = (
+        <InfoParagraph>
+          This guide configures your AWS OIDC policy to allow console access
+          with a script that you will run in AWS CloudShell.
+        </InfoParagraph>
+      );
+      prereqs = (
+        <InfoUl>
+          <li>{requiredAwsAccount}</li>
+          <li>List of AWS roles you want to be able to authenticate as.</li>
+        </InfoUl>
+      );
+      break;
+
+    case DiscoverGuideId.ServerAwsEc2Ssm:
+      links = {
+        systemsFleetManager: {
+          title: 'AWS Systems/Fleet Manager',
+          href: 'https://docs.aws.amazon.com/systems-manager/latest/userguide/fleet-manager.html',
+        },
+        ssmAgent: {
+          title: 'SSM Agent',
+          href: 'https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent-status-and-restart.html',
+        },
+        amazonSSMManagedInstanceCore: {
+          title: 'AmazonSSMManagedInstanceCore policy',
+          href: 'https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonSSMManagedInstanceCore.html',
+        },
+      };
+      overview = (
+        <InfoParagraph>
+          This guide is used to enroll <i>all</i> EC2 instances controlled by{' '}
+          <InfoExternalTextLink href={links.systemsFleetManager.href}>
+            AWS Systems/Fleet Manager
+          </InfoExternalTextLink>{' '}
+          in a region.
+        </InfoParagraph>
+      );
+      prereqs = (
+        <InfoUl>
+          <li>{requiredAwsAccount}</li>
+          <li>
+            List of OS users you want to be able to connect as (i.e. root,
+            ubuntu, etc).
+          </li>
+          <li>
+            <InfoExternalTextLink href={links.ssmAgent.href}>
+              SSM Agent
+            </InfoExternalTextLink>{' '}
+            running in target EC2 instances, and have the
```

Comment:
This renders without a space between `the` and  AmazonSSMManagedInstanceCore 
![image](https://github.com/user-attachments/assets/e5703fc1-9f3e-47ed-b111-abea7c5fbc7f)